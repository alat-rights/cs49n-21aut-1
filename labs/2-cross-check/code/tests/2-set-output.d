2-set-output.o: 2-set-output.c test-helper.h ../rpi.h ../fake-pi.h
