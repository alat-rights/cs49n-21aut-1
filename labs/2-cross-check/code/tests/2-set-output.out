TRACE: calling pi code
TRACE: GET32(0x20200000) = 0x327b23c6
TRACE: PUT32(0x20200000) = 0x327b23c1
TRACE: GET32(0x20200000) = 0x327b23c1
TRACE: PUT32(0x20200000) = 0x327b23c9
TRACE: GET32(0x20200000) = 0x327b23c9
TRACE: PUT32(0x20200000) = 0x327b2249
TRACE: GET32(0x20200000) = 0x327b2249
TRACE: PUT32(0x20200000) = 0x327b2249
TRACE: GET32(0x20200000) = 0x327b2249
TRACE: PUT32(0x20200000) = 0x327b1249
TRACE: GET32(0x20200000) = 0x327b1249
TRACE: PUT32(0x20200000) = 0x32789249
TRACE: GET32(0x20200000) = 0x32789249
TRACE: PUT32(0x20200000) = 0x32649249
TRACE: GET32(0x20200000) = 0x32649249
TRACE: PUT32(0x20200000) = 0x32249249
TRACE: GET32(0x20200000) = 0x32249249
TRACE: PUT32(0x20200000) = 0x31249249
TRACE: GET32(0x20200000) = 0x31249249
TRACE: PUT32(0x20200000) = 0x9249249
TRACE: GET32(0x20200004) = 0x643c9869
TRACE: PUT32(0x20200004) = 0x643c9869
TRACE: GET32(0x20200004) = 0x643c9869
TRACE: PUT32(0x20200004) = 0x643c9849
TRACE: GET32(0x20200004) = 0x643c9849
TRACE: PUT32(0x20200004) = 0x643c9849
TRACE: GET32(0x20200004) = 0x643c9849
TRACE: PUT32(0x20200004) = 0x643c9249
TRACE: GET32(0x20200004) = 0x643c9249
TRACE: PUT32(0x20200004) = 0x643c9249
TRACE: GET32(0x20200004) = 0x643c9249
TRACE: PUT32(0x20200004) = 0x643c9249
TRACE: GET32(0x20200004) = 0x643c9249
TRACE: PUT32(0x20200004) = 0x64249249
TRACE: GET32(0x20200004) = 0x64249249
TRACE: PUT32(0x20200004) = 0x64249249
TRACE: GET32(0x20200004) = 0x64249249
TRACE: PUT32(0x20200004) = 0x61249249
TRACE: GET32(0x20200004) = 0x61249249
TRACE: PUT32(0x20200004) = 0x49249249
TRACE: GET32(0x20200008) = 0x66334873
TRACE: PUT32(0x20200008) = 0x66334871
TRACE: GET32(0x20200008) = 0x66334871
TRACE: PUT32(0x20200008) = 0x66334849
TRACE: GET32(0x20200008) = 0x66334849
TRACE: PUT32(0x20200008) = 0x66334849
TRACE: GET32(0x20200008) = 0x66334849
TRACE: PUT32(0x20200008) = 0x66334249
TRACE: GET32(0x20200008) = 0x66334249
TRACE: PUT32(0x20200008) = 0x66331249
TRACE: GET32(0x20200008) = 0x66331249
TRACE: PUT32(0x20200008) = 0x66309249
TRACE: GET32(0x20200008) = 0x66309249
TRACE: PUT32(0x20200008) = 0x66249249
TRACE: GET32(0x20200008) = 0x66249249
TRACE: PUT32(0x20200008) = 0x66249249
TRACE: GET32(0x20200008) = 0x66249249
TRACE: PUT32(0x20200008) = 0x61249249
TRACE: GET32(0x20200008) = 0x61249249
TRACE: PUT32(0x20200008) = 0x49249249
TRACE: GET32(0x2020000c) = 0x74b0dc51
TRACE: PUT32(0x2020000c) = 0x74b0dc51
TRACE: GET32(0x2020000c) = 0x74b0dc51
TRACE: PUT32(0x2020000c) = 0x74b0dc49
TRACE: pi exited cleanly: 135 calls to random
